ups/product_deps

```diff
@@ -34,9 +34,9 @@ fwdir   product_dir scripts
 # Add the dependent product and version
 
 product             version
-larsoft             v09_39_01
-sbnobj              v09_13_00
-sbnanaobj           v09_17_07
+larsoft             v09_37_01_01
```

Comment:
looks good, just be careful undoing these changes to the version numbers as you do the release